### PR TITLE
Fixing circle ci tests

### PR DESCRIPTION
**Pull request by m-242**
Tests still don't pass, but this isn't a CircleCI problem anymore.

### CODE REVIEW

**Comment by bbriggs** [comment]
Not quite working. Also, `go test` failing on no test files seems like new behavior to me.

**Comment by m-242** [comment]
Yeah I know this doesn't work.

They were in fact two problems : one was that the libs (modules ?) in use weren't dowloaded properly by CircleCI, and I fixed that in this PR.

The second problem was that there is an inconsistency between the implementation and the test of the links plugin.
This problem was fixed in PR#53

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,6 +14,19 @@ jobs:
     working_directory: /go/src/github.com/bbriggs/bitbot
     steps:
      - checkout
+     - run:
+        name: Get needed dependencies
+        command: |
+            go get "github.com/bbriggs/quotes/model"
```
Comment (bbriggs) [inline .circleci/config.yml]: We're using go modules, so make sure that Circle is using the correct version of Go (>=1.12) and then you should be able to `go run` and go will take care of deps on its own.

**Comment by bbriggs** [comment]
It's passing on Circle

**Pull request merged**